pycode/memilio-epidata/memilio/epidata/getDIVIData.py

```diff
@@ -208,7 +208,7 @@ def divi_data_sanity_checks(df=pd.DataFrame()):
     # check if size of dataframe is not unusal
     # data colletion starts at 24.04.2020
     num_dates = (date.today() - date(2020, 4, 24)).days
-    min_num_data = 390*num_dates  # not all 400 counties report every day
+    min_num_data = 380*num_dates  # not all 400 counties report every day
```

Comment:
Download fails now for 390 because of less counties reporting now. I changed it to 380 as minimum to make a small change but maybe this needs to be adjusted again in the fututre

Reply:
so, this sounds very heuristic. But you should also comment this in the code

Reply:
I added a TODO